docs/implementation-references/getting-started-guide.md

```diff
@@ -159,27 +213,40 @@ Expect the following line on successful DI completion.
 DI complete, GUID is <guid>
 ```
 
-***NOTE:***
+**After completion of DI, the FDO credentials are stored into `credential.bin` file. The credentials file include `rvinfo` from manufacturer, which is later used by device to contact RV server, once it is powered on at the client side. Finally, the initialized device is boxed and sold to customers.**
+
+!!!Additional_Configurations
         - Additional arguments for [configuring PRI device](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo/device#configuring-the-device-service).
         - Configuring PRI device for [HTTPS/TLS communication](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo/device#configuring-device-for-httpstls-communication).
         - [Read more](https://github.com/secure-device-onboard/pri-fidoiot/blob/master/component-samples/demo/device/README.md) about Device Intialization.
 
 
-3.&nbsp;Voucher Extension & TO0 for PRI Device
+#####Voucher Extension & TO0 for PRI Device
 
 <figure>
+  <center>
   <img src="../../images/slide3.png"/>
   <figcaption>Voucher extension</figcaption>
+  </center>
 </figure>
+<br>
 
-***During TO0, the FDO Owner identifies itself to Rendezvous Server, establishes the mapping of GUID to the Owner IP address. TO0 ends with RV Server having an entry in a table that associates the Device GUID with the Owner Service’s rendezvous 'blob.'***
+**During TO0, the FDO Owner identifies itself to Rendezvous Server, establishes the mapping of GUID to the Owner IP address. TO0 ends with RV Server having an entry in a table that associates the Device GUID with the Owner Service’s rendezvous 'blob.'**
```

Comment:
identifies itself to RV "and" established mapping

Reply:
Do we also have to say where to find connection string for database

Reply:
Updated. What do you mean by connection string?